pkg/restore/restore.go

```diff
@@ -75,27 +74,31 @@ type kubernetesRestorer struct {
 // prioritizeResources takes a list of pre-prioritized resources and a full list of resources to restore,
 // and returns an ordered list of GroupResource-resolved resources in the order that they should be
 // restored.
-func prioritizeResources(mapper meta.RESTMapper, priorities []string, resources []*metav1.APIResourceList) ([]schema.GroupResource, error) {
+func prioritizeResources(helper discovery.Helper, priorities []string, includedResources *collections.IncludesExcludes) ([]schema.GroupResource, error) {
 	var ret []schema.GroupResource
 
 	// set keeps track of resolved GroupResource names
 	set := sets.NewString()
 
 	// start by resolving priorities into GroupResources and adding them to ret
 	for _, r := range priorities {
-		gr := schema.ParseGroupResource(r)
-		gvr, err := mapper.ResourceFor(gr.WithVersion(""))
+		gr, err := helper.ResolveGroupResource(r)
 		if err != nil {
 			return nil, err
 		}
-		gr = gvr.GroupResource()
+
+		if !includedResources.ShouldInclude(gr.String()) {
+			glog.Infof("Not including resource %s", gr.String())
```

Comment:
If gr supports String() you can just do this
glog.Infof("Not including resource %v", gr)

nit: should this be a debug not an info?

Reply:
Thx for the tip.

We're going to revisit all our logging when we switch to logrus.

Reply:
@davecheney not sure if we're doing something wrong, but with the change so we are no longer explicitly invoking String(), it doesn't actually call our String(). So we get this:

```
Using default restorer for { secrets}
```

instead of this:

```
Using default restorer for secrets
```

Reply:
argh. I wonder if this is a glog issue - I tested this out with fmt.Printf in a test project before switching to ensure it worked the way I thought, but didn't actually test it with glog.

Reply:
Actually this is with my custom "logger"

Reply:
actually, the problem is that String() has a pointer receiver but gr is not a pointer.